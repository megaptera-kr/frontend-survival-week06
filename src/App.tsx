import FilterableRestaurants from './components/filterableRestaurants/FilterableRestaurants';
import Cart from './components/cart/Cart';

export default function App() {
  return (
    <main>
      <h1>Kiosk week 6</h1>
      <section style={{ display: 'flex' }}>
        <FilterableRestaurants />
        <Cart />
      </section>
    </main>
  );
}
