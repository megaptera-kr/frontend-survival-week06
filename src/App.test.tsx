import { render, screen } from '@testing-library/react';

import App from './App';

test('App', async () => {
  const { asFragment } = render(<App />);

  await screen.findByText('짜장면');

  expect(asFragment()).toMatchSnapshot();
});
