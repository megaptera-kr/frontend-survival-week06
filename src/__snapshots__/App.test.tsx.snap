// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App 1`] = `
<DocumentFragment>
  <main>
    <h1>
      Kiosk week 6
    </h1>
    <section
      style="display: flex;"
    >
      <article>
        <h2>
          식당 메뉴표
        </h2>
        <div>
          <div>
            <label
              for="search-restaurant-name"
            >
              search
            </label>
            <input
              id="search-restaurant-name"
              name="searchText"
              value=""
            />
          </div>
          <ul
            style="display: flex; list-style: none; padding: 0px;"
          >
            <li>
              <label>
                <input
                  checked=""
                  name="category"
                  type="radio"
                  value="전체"
                />
                전체
              </label>
            </li>
            <li>
              <label>
                <input
                  name="category"
                  type="radio"
                  value="중식"
                />
                중식
              </label>
            </li>
            <li>
              <label>
                <input
                  name="category"
                  type="radio"
                  value="한식"
                />
                한식
              </label>
            </li>
            <li>
              <label>
                <input
                  name="category"
                  type="radio"
                  value="일식"
                />
                일식
              </label>
            </li>
          </ul>
        </div>
        <table>
          <caption>
            식당 목록
          </caption>
          <thead>
            <tr>
              <th>
                카테고리
              </th>
              <th>
                식당 명
              </th>
              <th>
                메뉴
              </th>
            </tr>
          </thead>
          <tbody>
            <tr>
              <td>
                <span>
                  중식
                </span>
              </td>
              <td>
                <span>
                  메가반점
                </span>
              </td>
              <td>
                <ul>
                  <li>
                    <span>
                      짜장면
                    </span>
                    <span>
                      8000
                    </span>
                    <button
                      aria-label="짜장면 추가"
                      type="button"
                    >
                      장바구니 추가
                    </button>
                  </li>
                  <li>
                    <span>
                      짬뽕
                    </span>
                    <span>
                      5000
                    </span>
                    <button
                      aria-label="짬뽕 추가"
                      type="button"
                    >
                      장바구니 추가
                    </button>
                  </li>
                </ul>
              </td>
            </tr>
            <tr>
              <td>
                <span>
                  한식
                </span>
              </td>
              <td>
                <span>
                  메리김밥
                </span>
              </td>
              <td>
                <ul>
                  <li>
                    <span>
                      김밥
                    </span>
                    <span>
                      3000
                    </span>
                    <button
                      aria-label="김밥 추가"
                      type="button"
                    >
                      장바구니 추가
                    </button>
                  </li>
                </ul>
              </td>
            </tr>
            <tr>
              <td>
                <span>
                  일식
                </span>
              </td>
              <td>
                <span>
                  혹등고래카레
                </span>
              </td>
              <td>
                <ul>
                  <li>
                    <span>
                      카레
                    </span>
                    <span>
                      10000
                    </span>
                    <button
                      aria-label="카레 추가"
                      type="button"
                    >
                      장바구니 추가
                    </button>
                  </li>
                </ul>
              </td>
            </tr>
          </tbody>
        </table>
      </article>
      <article>
        <h2>
          장바구니
        </h2>
        <div
          style="display: flex; column-gap: 8px;"
        >
          <div>
            <p>
              총 주문 개수: 0개
            </p>
            <p>
              총 주문 금액: 0원
            </p>
          </div>
          <button
            type="button"
          >
            비우기
          </button>
          <button
            type="button"
          >
            주문하기
          </button>
        </div>
        <ul />
      </article>
    </section>
  </main>
</DocumentFragment>
`;
